### PR TITLE
Fix checkbox change editor focus

### PR DESCRIPTION
**Pull request by Komediruzecki**
Add scroll to the current line instead of the top of the doc

Fixes: #56 

Showcase:

https://user-images.githubusercontent.com/18196945/156926739-05cc9973-1e06-46e5-9bbf-c789cd61fa64.mp4

### CODE REVIEW

File: src/components/organisms/LocalSearch.tsx [hunk 1]
```diff
@@ -51,6 +51,15 @@ interface LocalSearchProps {
 
 export type SearchResultNavigationDirection = 'next' | 'previous'
 
+export function scrollEditorToLine(
```
Comment (Rokt33r) [inline src/components/organisms/LocalSearch.tsx:54]: Extract method from here. This method is now used for other purposes than local searching.

File: src/components/organisms/NoteDetail.tsx [hunk 2]
```diff
@@ -227,6 +234,10 @@ class NoteDetail extends React.Component<NoteDetailProps, NoteDetailState> {
         }
       },
       () => {
+        if (this.codeMirror != null && checkboxUpdate) {
+          this.codeMirror.focus()
+          this.setInitialCursor()
```
Comment (Rokt33r) [inline src/components/organisms/NoteDetail.tsx:239]: Initial cursor is not used for this purpose. Please make a new method.

File: src/components/organisms/NoteDetail.tsx [hunk 3]
```diff
@@ -214,7 +220,8 @@ class NoteDetail extends React.Component<NoteDetailProps, NoteDetailState> {
   }
 
   updateContent = (
-    newValueOrUpdater: string | ((prevValue: string) => string)
+    newValueOrUpdater: string | ((prevValue: string) => string),
+    checkboxUpdate = false
```
Comment (Rokt33r) [inline src/components/organisms/NoteDetail.tsx:224]: Use other name to explain what behavior would change from this argument.

**Review by Rokt33r — CHANGES_REQUESTED** [review]

File: src/components/organisms/LocalSearch.tsx (hunk 1 above)
Comment (Komediruzecki) [reply src/components/organisms/LocalSearch.tsx:54]: Done

**Pull request merged**